### PR TITLE
Pull request to review the code for the Bootstrap project

### PR DESCRIPTION
**Pull request by Prabhakarzx**
# This is to request the code review for our project named *use bootstrap* :checkered_flag: 

**This webpage is the clone of _[Newsweek](https://www.newsweek.com/)_ site.** :globe_with_meridians: 

## In this project we've used three main files :file_folder: 
- Bootstrap Link
- HTML(index.html)
- CSS(style.css)

## In this Project we've used **HTML, CSS** and **Bootstrap Framework** :man_technologist:  :woman_technologist: 

**The above mentioned site is cloned for educational purpose only. All rights are reserved to its original owner(s).** :copyright: 

## **Thank You for taking your time to review our code.** :bouquet: 

# **_Thanks!_**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1619 @@
...
+  <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap" rel="stylesheet">
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" />
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+  <div class="container">
```
Comment (peterrobert) [inline index.html]: ### [OPTIONAL]

I assume this class is to be applied to the whole code. If that is the case there is no need to add a div element just add the class to the body tag

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 
### HELLO @Prabhakarzx :wave:  AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT. AWESOME WORK FOR A FIRST REVIEW :clap: 

**HOWEVER BEFORE YOU CAN CONTINUE LET'S MAKE THE FOLLOWING CHANGES** :arrow_down: 

- [x] Good job creating a very responsive web page :clap:  However, your web page should have padding on both the left and the right. Your project only has padding on the right.  Also at the large screen, the webpage should be centered on the screen.
### YOURS
padding
### ORIGINAL
padding original

- [x] You have used bootstrap cards for your project :clap: However, You need to remove the card borders. This will improve your project and get it a step closer to resemble the original

remove borders among cards

- [x] You have done a good job on your footer. Great job using the footer semantic tag. However, let's improve it by making your menu links resemble the original.

### YOURS
footer bootsrap
### ORIGINAL
footer bootsrap original

- [x] The mobile menu bar is well responsive :clap: However, at 350px your menu items seem to fall off their container. This should not be the case. Improve on this. 
**Hint** you can use flexbox for that.

350px menu

- [ ] Your webpage has met all the required sections. However, the mobile view of the more stories section does not resemble the original. Kindly fix that.
### YOURS
more a=stories n
### ORIGINAL
more stories mobile view original

### [OPTIONAL]
There some CSS warnings in your CSS file. You can use [this](https://jigsaw.w3.org/css-validator/) tool to validate and fix them.

### Great work with your readme.

kindly fix all the above and submit your project for another review.

**if you have any question feel free to reach out to me on slack** @peterrobert

### HAPPY CODING :computer:

**Comment by Prabhakarzx** [comment]
# Dear @peterrobert Thank You so much for the detailed review! :100: 
- [x] We've fixed all the issues.
- [x] We learned even more while browsing to fix the screen size issue.

## Thank You! :bouquet:

**Review by expjazz — APPROVED** [review]
## Hey Team
### Congratulations! The project is approved. Great job implementing the changes requested! :clap: 

There is just one detail that would make the project improve, but this is optional since the project has already met the requirements.
[optional] => on the mobile screen size there is no padding. You can also work on the blue background between sections.

#### original :arrow_down: 

Screen Shot 2020-07-09 at 16 03 05

#### Your clone

Screen Shot 2020-07-09 at 16 03 47

### Good Luck on the next projects and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

**Comment by Prabhakarzx** [comment]
# Dear @expjazz Thank you so much for taking your time, reviewing and approving our project. :bouquet:
## We're happy to see that optional change and
- [x] we implemented that suggested change too! :100: 
**Thank You so much for all of your best wishes!** :bouquet: 
# Thanks! :bouquet:

**Pull request merged**